uint8 frame_id
uint8 followed_data_length
int16 angular_velocity_x 
int16 angular_velocity_y 
int16 angular_velocity_z